README.md

```diff
@@ -1,3 +1,28 @@
 # Build Server Protocol for IntelliJ IDEA
 
-**WORK IN PROGRESS** stand-alone plugin support for the [Build Server Protocol](https://build-server-protocol.github.io/) in IntelliJ IDEA. The goal of this plugin is to replace the BSP support bundled with the Scala plugin for IntelliJ IDEA.
\ No newline at end of file
+**WORK IN PROGRESS**
+
+Stand-alone plugin support for the [Build Server Protocol](https://build-server-protocol.github.io/) in IntelliJ IDEA. The goal of this plugin is to replace the BSP support bundled with the [Scala plugin for IntelliJ IDEA](https://github.com/JetBrains/intellij-scala).
+
+This project is built using plugin [sbt-idea-plugin](https://github.com/JetBrains/sbt-idea-plugin).
+
+### Requirements
+
+To be able to develop this, you need:
+- IntelliJ IDEA version matching the version set in `build.sbt` file, e.g.
```

Comment:
Not strictly necessary, generally the latest release version should be sufficient.